.github/workflows/build-and-test.yml

```diff
@@ -95,6 +98,8 @@ jobs:
           git clone --branch develop https://github.com/precice/precice.git precice-core
           mkdir -p precice
           cp precice-core/src/precice/SolverInterface.hpp precice/SolverInterface.hpp
+          cp precice-core/src/precice/Tooling.hpp precice/Tooling.hpp
+          cp precice-core/src/precice/Tooling.hpp precice/Tooling.cpp
```

Comment:
Does this belong into this PR? Is this not more a general compatibility thing for the testing?

Reply:
This change is indeed not part of the main change in this pull request, but has been included here to get the tests to work. The tests are currently breaking on develop anyway, so pushing changes separately leads the tests to break on practically every feature branch.

Reply:
Thanks for the suggestions, its done now :+1: